src/common/components/contactInfo/contactInfo.component.spec.js

```diff
@@ -228,8 +310,8 @@ describe('contactInfo', function () {
         emailFormUri: '/emails/crugive',
         'donor-type': 'Staff'
       })
-
       expect(self.controller.orderService.addEmail).toHaveBeenCalledWith('someone@asdf.com', '/emails/crugive')
+      expect(self.controller.orderService.storeRadioStationData).toHaveBeenCalledWith(radioStationData)
```

Comment:
It seems like this should be split out into a separate test `it('should save radio station')`